core/services/relay/evm/capabilities/workflows/syncer/workflow_syncer_test.go

```diff
@@ -151,7 +152,7 @@ func Test_EventHandlerStateSync(t *testing.T) {
 	require.Eventually(t, func() bool {
 		numEvents := len(testEventHandler.GetEvents())
 		return numEvents == numberWorkflows
-	}, 5*time.Second, time.Second)
+	}, tests.WaitTimeout(t), 100*time.Millisecond)
```

Comment:
Can we leave these at one second? Going faster creates CPU pressure (10x) for minor tradeoff (completing 0.9s sooner).
```suggestion
	}, tests.WaitTimeout(t), time.Second)
```

Reply:
Changed back - 280602f92666ece4ca427b34275eb52fb98ff8e4